jetty-util/src/main/java/org/eclipse/jetty/util/Pool.java

```diff
@@ -197,32 +235,14 @@ public Entry acquireAt(int idx)
     /**
      * Acquire an entry from the pool.
      * Only enabled entries will be returned from this method and their enable method must not be called.
+     * The implementation first tries the pool strategy and then a brute force iteration over entries.
```

Comment:
This comment isn't accurate anymore as the iteration is also part of the strategy.